### DISCUSSION TITLE
Welcome to IDDconf workshop 2!

### DISCUSSION THREAD

**Discussion by avallecam** [epiverse-trace/research-compendium#1] [Announcements]
## 👋 Welcome!

To get started, comment below with your:

- Name / Pronouns / What is your most recent favourite song, music artist or band?

**Comment by benjaminjohnsinger** [comment]
Ben, He/Him, Bug Like an Angel by Mitski

**Comment by p-ruiz-cuenca** [comment]
Hello! My name is Pablo, my pronouns are he/him. I have been re-visiting Taylor Swifts greatest hits in preparation for the Eras Tour (next year....)

**Comment by emjnixon15** [comment]
Hi, my name is Emily and my pronouns are she/her. Like Pablo I have been listening to too much Taylor Swift after going to the Eras Tour earlier this year.

**Reply by SMAHSMAH** [reply]
Michael (He/Him),
I speak the name of Jesus (Most recent favourite song, by Charity Gayle)

**Comment by scarabel** [comment]
Francesca, she/her, random songs from italian vintage artists

**Comment by mbkoltai** [comment]
Hi,
I'm Mihaly (pronounce: me-huy), pronouns he/him. A song I like is 'Just Like Honey' (The Jesus And Mary Chain).

**Comment by evandrokonzen** [comment]
Evandro; he/him; Guns N' Roses

**Comment by NiyeloStar1** [comment]
Hi, my name is Gbeminiyi and my pronouns are HE/HIM. I like football and I support Arsenal FC in England and 3SC FC in NIgeria.

**Comment by lauraguzmanrincon** [comment]
Laura, She/Her, old bands like Cult

**Comment by samuel-a-sutherland** [comment]
Hi, I'm Sam (he/him). I've been listening to Maisie Peters' new album a lot recently.

**Comment by SHVickers94** [comment]
Stephen, He/Him, Oliver Tree

**Comment by lorenzo-pellis** [comment]
Lorenzo, he/him, banana boat song

**Comment by emprestige** [comment]
Em, they/them, been obsessed with Noah Kahan lately

**Comment by ghost** [comment]
Hello, Hannah (she/her)
Mnike, Dj Maphorisa is currently on repeat :)
